bin/tass

```diff
@@ -78,17 +82,27 @@ when 'audit'
   opts = Trollop.options do
     usage 'audit'
     synopsis 'This command compares local configuration files for a given cluster to the existing launch configuration and autoscaling group running in AWS.'
-    opt :filename, 'Specify config file to load', type: :string, required: true
-    opt :config_dir, 'Specify the directory for configuration files', type: :string, short: :none, default: Tapjoy::AutoscalingBootstrap.config_dir
-    opt :env, 'Specify which environment config to load', required: true, type: :string
-    opt :prompt, 'Enable/disable prompts', default: true
+    common_args
   end
 
-  config, aws_env, user_data = Tapjoy::AutoscalingBootstrap::Base.new.configure_environment(opts[:filename], opts[:env], opts[:config_dir])
-  Aws.config[:region] = config[:aws_region]
+  config, aws_env, user_data = configure_environment(opts)
 
   config.merge!(aws_env.merge(user_data: Base64.encode64("#{user_data}")))
   Tapjoy::AutoscalingBootstrap::Audit.new(config)
+when 'scale'
```

Comment:
We'd discussed potentially having min/max stored in the yaml file. Desired probably ought to remain a CLI option - maybe include the ability to do it either way?